pages/admin/login/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { FormEvent, useState } from "react";
+import { NextPage } from "next";
+import { useRouter } from "next/router";
+
+const Login: NextPage = () => {
+  const [loginState, setLoginState] = useState({
+    username: "",
+    password: "",
+  });
+
+  const router = useRouter();
+
+  const login = () => {
+    fetch("http://localhost:5000/graphql", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        query: `
+          mutation login($email: String!, $password: String!) {
+            login(email: $email, password: $password) {
+              firstName
+            }
+          }
+        `,
```

Comment:
This is good,  but can we import this mutation from a `graphql/queries` folder. That way we can centralize all of our mutations.